qiskit-superstaq/qiskit_superstaq/serialization.py

```diff
@@ -120,13 +120,13 @@ def _assign_unique_inst_names(circuit: qiskit.QuantumCircuit) -> qiskit.QuantumC
 def serialize_circuits(
     circuits: Union[qiskit.QuantumCircuit, Sequence[qiskit.QuantumCircuit]]
 ) -> str:
-    """Serialize QuantumCircuit(s) into a single string
+    """Serialize QuantumCircuit(s) into a single string.
 
     Args:
-        circuits: a QuantumCircuit or list of QuantumCircuits to be serialized
+        circuits: A QuantumCircuit or list of QuantumCircuits to be serialized.
 
     Returns:
-        str representing the serialized circuit(s)
+        A str representing the serialized circuit(s).
```

Comment:
A bit out of date on docstring format decisions but would this apply?
```suggestion
        A `str` representing the serialized circuit(s).
```

Reply:
I would think so (I have been using backticks for the `True` and `False` in docstrings if stated as bool values). But also, I have been replacing str -> string in most cases.

Reply:
I would prefer string here.